go.mod

```diff
@@ -17,3 +17,65 @@ require (
 	golang.org/x/oauth2 v0.0.0-20211104180415-d3ed0bb246c8
 	golang.org/x/text v0.3.7
 )
+
+require (
```

Comment:
I don't understand `go.mod` enough to know why this is suddenly necessary.

Reply:
Do you mean additional require block? If so;

>  Because the number of explicit requirements may be substantially larger in an expanded Go 1.17 go.mod file, the newly-added requirements on indirect dependencies in a go 1.17 module are maintained in a separate require block from the block containing direct dependencies. 

sauce: https://go.dev/doc/go1.17#Tools